packages/labs/react/src/create-component.ts

```diff
@@ -354,6 +356,20 @@ the element.`);
 
         props[k] = v;
       }
+
+      // If component is to be server rendered with `@lit-labs/ssr-react` pass
+      // element properties in a special bag to be set by the server-side
+      // element renderer.
+      if (
+        NODE_MODE &&
+        createElement.name === 'litPatchedCreateElement' &&
```

Comment:
Could `litPatchedCreateElement` become mangled or minified and break this check?

Reply:
Since the `labs/ssr-react` package is not minified by us, and is meant to be used server-side, I think in practice this is unlikely to be mangled.

There is another potential failure point of this detection that users might wrap `createElement` with another enhancement on top, which we technically allow but we could add a note to make sure ours is the last wrapper. Another consideration would be to set a global flag rather than doing a function name check like this. My feeling is this is a premature concern but I'm open to ideas.

Reply:
Discussed offline - this sounds great. And global flag can be used down the line if required.